plugins/rest-api-endpoints/routes.json

```diff
@@ -2503,6 +2500,7 @@
       "url": "/orgs/:org/blocks/:username"
     },
     "update": {
+      "deprecated": "octokit.orgs.update() is deprecated, see https://developer.github.com/v3/orgs/#edit-an-organization",
```

Comment:
This deprecation is incorrect. Only the `members_allowed_repository_creation_type` parameter is deprecated